src/lib/list/selection-list.spec.ts

```diff
@@ -451,6 +481,13 @@ class SelectionListWithDisabledOption {
 }
 
 @Component({template: `
+<mat-selection-list>
+  <md-list-option [selected]="true">Item</md-list-option>
+</mat-selection-list>
```

Comment:
The template should have a two space indentation here (as in other test components)

Reply:
Done